src/style.css

```
@@ -0,0 +1,82 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: AvenirHeavy;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: bold;
+}
```

You should connect the fonts to te same font-family, [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) 7th item
```suggestion
@font-face {
  font-family: "Avenir";
  src: url(./fonts/Avenir-Book.ttf);
  font-style: normal;
  font-weight: 300;
}

@font-face {
  font-family: "Avenir";
  src: url(./fonts/Avenir-Heavy.ttf);
  font-style: normal;
  font-weight: bold;
}
```